charts/graylog/templates/statefulset.yaml

```diff
@@ -72,12 +73,14 @@ spec:
               rm -rf /usr/share/graylog/data/journal/messagejournal-0
               rm -rf /usr/share/graylog/data/journal/recovery-point-offset-checkpoint
             {{- end }}
+            {{- if semverCompare "< 4.2.0-0" ( $graylogVersion ) }}
```

Comment:
The Graylog 4 is very out of date. We can say the chart no longer supports Graylog version < 4.2.
You can remove all `kubectl` related.